docs/man/containerd-config.8.md

```diff
@@ -10,22 +10,31 @@ containerd config [command]
 
 ## DESCRIPTION
 
-The *containerd config* command has one subcommand, named *default*, which
+The _containerd config_ command has one subcommand, named _default_, which
```

Comment:
Not just one subcommand?

Reply:
sorry i just changed the options. i didn't read the description.